Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/SettingViewHolder.java

```diff
@@ -57,11 +55,10 @@ protected static void showNotRuntimeEditableError()
   }
 
   /**
-   * Gets handles to all this ViewHolder's child views using their XML-defined identifiers.
-   *
-   * @param root The newly inflated top-level view.
+   * Gets handles to all this ViewHolder's child views using their view bindings. Must be called
+   * in the constructor of the class that implements a SettingViewHolder.
    */
-  protected abstract void findViews(View root);
+//  protected abstract void findViews();
```

Comment:
Please remove this instead of just commenting it out.

Reply:
My bad